### DISCUSSION TITLE
Order by which OCSP, CDP are checked

### DISCUSSION THREAD

**Discussion by amdrsantos** [strongswan/strongswan#1233] [Q&A] [answered]
<name>.crl_uris and <name>.ocsp_uris are comma-separated list of URIs.
The order by which the servers are checked is the order given in the comma-separated list?
Example:
For
`authorities.my_auth1.ocsp_uris = http://server1.test, http://server2.test, http://server3.test`
the order by which servers are checked is first server1, then server2 and then server3.
Can you confirm?
Thanks, Alex

**Comment by tobiasbrunner** [comment, marked as answer]
Yes.
